soliket/clusters/survey.py

```diff
@@ -169,7 +169,7 @@ def __init__(
                 self.filetile, dtype=np.str, usecols=0, unpack=True
             )
             self.tilearea = np.loadtxt(
-                self.filetile, dtype=np.float, usecols=1, unpack=True
+                self.filetile, dtype=float, usecols=1, unpack=True
```

Comment:
Hmm... I will ask about this as it seems un-necessary as everything was fine before and it is not good to touch other likelihoods when deadling with mflike here.

Reply:
ok. This is a python 3.10.9 error (it was just deprecated for previous versions). Maybe it is worth to open an issue just to remember about the problem?

Reply:
I reverted the correction here